app/src/main/java/net/skyscanner/backpack/demo/ui/CasesScreen.kt

```diff
@@ -0,0 +1,93 @@
+package net.skyscanner.backpack.demo.ui
+
+import android.content.Intent
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import net.skyscanner.backpack.compose.horizontalnav.BpkHorizontalNav
+import net.skyscanner.backpack.compose.horizontalnav.BpkHorizontalNavTab
+import net.skyscanner.backpack.compose.icon.BpkIcon
+import net.skyscanner.backpack.compose.navigationbar.BpkTopNavBar
+import net.skyscanner.backpack.compose.navigationbar.IconAction
+import net.skyscanner.backpack.compose.navigationbar.NavIcon
+import net.skyscanner.backpack.compose.tokens.Settings
+import net.skyscanner.backpack.demo.R
+import net.skyscanner.backpack.demo.SettingsActivity
+import net.skyscanner.backpack.demo.compose.ComponentItem
+import net.skyscanner.backpack.demo.meta.Component
+import net.skyscanner.backpack.demo.meta.Story
+import net.skyscanner.backpack.demo.meta.all
+
+@Composable
+fun CasesScreen(
```

Comment:
It shows not only tabs but lists of cases as well. Haven't found better naming :)

Reply:
I'd say ultimately this is the `ComponentScreen` - `Cases` is very unclear what that means. particularly as it could just be the component details directly